packages/transitive-overlay/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "composite": true,
-    "lib": ["es2019"],
+    "lib": ["es2019", "dom"],
```

Comment:
What is this change for?

Reply:
`window` gets marked as an unknown variable without it